addons/mail/models/mail_message.py

```diff
@@ -121,7 +121,7 @@ def _get_default_author(self):
     need_moderation = fields.Boolean('Need moderation', compute='_compute_need_moderation', search='_search_need_moderation')
     #keep notification layout informations to be able to generate mail again
     layout = fields.Char('Layout', copy=False)  # xml id of layout
-    layout_values = fields.Char('Notification values', copy=False)
+    add_sign = fields.Boolean(default=False)
```

Comment:
Ok, we will have to note for migration team that old messages should not have add_sign to True. Old messages that people will resend will not have the signature but that's ok.

Default value is True by the way.